drivers/SmartThings/matter-hrap/src/init.lua

```diff
@@ -0,0 +1,129 @@
+-- Copyright 2025 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local MatterDriver = require "st.matter.driver"
+local clusters = require "st.matter.clusters"
+
+local utils = require "lustre.utils"
+local log = require "log"
+
+local capabilities = require "st.capabilities"
+local threadVersion = capabilities["threadVersion"]
+local routerState = capabilities["routerState"]
+local routerName = capabilities["routerName"]
+local wifiSsid = capabilities["wifiSsid"]
+
+-- Include driver-side definitions when lua libs api version is <13
+local version = require "version"
+if version.api < 13 then
+  clusters.ThreadBorderRouterManagement = require "ThreadBorderRouterManagement"
+  clusters.WifiNetworkMangement = require "WiFiNetworkManagement"
+end
+
+
+--[[ ATTRIBUTE HANDLERS ]]--
+
+local function ssid_attribute_handler(driver, device, ib)
+  if ib.data.value == string.char(0x014) then -- TLV-encoded NULL
+    log.info("Ssid is null. Per the spec, no primary Wi-Fi network is available at the moment.")
```

Comment:
as a rule of thumb, I would suggest using `device.log` instead of `log` just because its essentially the same thing but it prefixes the log message with the device ID for easier debugging